initfiles/componentfiles/configschema/xsd/sparkthor.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+        xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+        xmlns:hpcc="someuri">
+    <xs:include schemaLocation="types.xsd"/>
+    <hpcc:insert hpcc:schemaPath="/Environment/Software">
+        <xs:element name="SparkThorProcess" hpcc:itemType="sparkthor" hpcc:displayName="Spark Thor" maxOccurs="1" hpcc:class="component"
```

Comment:
I'm not sure how it appears in the current xsd schema, but the display name should be "SparkThor" or "Spark-Thor"